File: codeforces/2030A.cpp
#include <bits/stdc++.h>
using namespace std;
typedef long long ll;

int main(){
    ll t;
    cin>>t;
    while(t--){
        ll n;cin>>n;
        vector<ll> v(n);
        for(ll i=0;i<n;i++)cin>>v[i];
        ll maxi=0, mini=1000000;
        for(ll i=0;i<n;i++){
            ll x=v[i];
            maxi=max(maxi,x);
            mini=min(mini,x);
        }
        cout<<((maxi-mini)*(n-1))<<"\n";
    }
}

File: codeforces/2030A.py
t = int(input())
while t>0 :
    t-=1
    n= int(input())
    maxi=0
    mini=1000000
    a =input().split()
    for i in range(n):
        x= int(a[i])
        maxi=max(maxi,x)
        mini=min(mini,x)
    print((maxi-mini)*(n-1)) 